src/constants.ts

```diff
@@ -1,3 +1,11 @@
 export const ReadiumWebpubContext = 'http://readium.org/webpub/default.jsonld';
 
 export const IS_DEV = process.env.NODE_ENV === 'development';
+
+// we have to set a constant height to make this work with R2D2BC
+export const HEADER_HEIGHT = 48;
```

Comment:
new ticket:  How do we make this stuff configurable?

Reply:
Yea good point for a new ticket. Wonder where I should put that one

Reply:
Probably some kind of config object?  I was thinking about this when trying to make `defaultScrolling` configurable - we'll need to figure out a way have the app consume something like this, cuz I don't think it's tenable to put them in props forever.